decidim-admin/app/packs/src/decidim/admin/controllers/slug/slug.test.js

```diff
@@ -0,0 +1,239 @@
+/* global jest */
+
+import { Application } from "@hotwired/stimulus"
+import SlugController from "./controller"
```

Comment:
Shall we resolve this warning?

Reply:
This particular warning is caused by the fact jest cannot resolve the controller path. I would like to merge it as is (being just a lint warning), and handle later in the process.